### PR TITLE
PP-402: remove netlibtoc.lib liblmx-altair.lib dependency from the windows project files

### PR DESCRIPTION
**Pull request by lcnja**
#### Issue-ID

* **[PP-402](https://pbspro.atlassian.net/browse/PP-402)**

#### Problem description
* Error 319 fatal error LNK1181: can not open input file "netlibtoc.lib" "liblmx-altair.lib"

#### Cause / Analysis
* Incorrect Dependencies

#### Solution description
* Kindly remove Incorrect Dependencies from the windows project files.

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [x] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [ ] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [ ] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

…ndows project files

### CODE REVIEW

File: src/cmds/scripts/pbs_migration_assistant.bat [hunk 1]
```diff
@@ -15,28 +15,15 @@ echo off
 SETX /m PBS_CONF_FILE "%WINDIR%\Temp\PBS Pro Backup\pbs.conf"
 echo In Backup Mode...
 echo ### Backing up Files...
-%backupcmd% "%pbs_exec_dir%\lib\liblmx-altair.dll" "%backupdir%MyDLLBackUp"
```
Comment (gajendrasharmagit) [inline src/cmds/scripts/pbs_migration_assistant.bat]: Thanks for the update. On second thoughts, I agree with earlier change of removing this file altogether...:-) The file was originally designed to automate the migration upgrade steps on Windows platform but in it's current form, it is a dummy.

**Comment by lcnja** [comment]
Let me delete this file and resubmit ,ok?

**Comment by gajendrasharmagit** [comment]
ok

File: src/cmds/scripts/pbs_migration_assistant.bat (hunk 1 above)
Comment (gajendrasharmagit) [inline src/cmds/scripts/pbs_migration_assistant.bat]: This will require a documentation update too. Please check the checklist.

**Comment by lcnja** [comment]
Is that ok now?

**Comment by gajendrasharmagit** [comment]
Please start the documentation here(https://pbspro.atlassian.net/wiki/display/PD) for removal of pbs_migration_asistant.bat. Steps to do migration upgrade on Windows, will change.

**Comment by gajendrasharmagit** [comment]
Please go through Section 7.10: Upgrading under Windows of PBS Installation guide:
http://www.pbsworks.com/pdfs/PBSProInstallGuide13.1.pdf
This document needs to be updated to address removal of pbs_migration_assistant.bat.

**Comment by mike0042** [comment]
This pull request is out of date with master and contains unsigned commits. Please update your branch and sign your commit.

**Comment by subhasisb** [comment]
@lcnja can you kindly update your pull request. If you need assistance with the documentation work, kindly let us know and somebody else might be able to help you.

**Comment by lcnja** [comment]
I don't know how&where to update the documentation,please help me,thx

**Comment by subhasisb** [comment]
Thanks @lcnja some of us will help you with the documentation - over time. Meantime, can you kindly update and sign your commit?

**Comment by lcnja** [comment]
please close this one,I have created a new pull request [#147](https://github.com/PBSPro/pbspro/pull/147)

**Comment by subhasisb** [comment]
Closing per request from @lcnja

**Pull request closed without merging**